specs/pvs/corla.pvs

```diff
@@ -1,55 +1,61 @@
-%%% of the system delivered to the Colorado Department of State. The prose
-%%% annotations should help Colorado Department of State elections domain experts
-%%% and technical staff to assess the completeness and correctness of the
-%%% ColoradoRLA system/tool.
-%%%
-%%% This document is written in a literate style using the PVS
-%%% theorem prover. Writing system specifications in this fashion is
-%%% called (formal) domain engineering. Our formal domain model is
-%%% written in PVS's higher-order logic (HOL). Our informal domain
-%%% model is written in the Free \& Fair System Specification Language,
-%%% or FAFESSL for short. FAFESSL is a daughter of Extended BON which
-%%% was, in turn, a daughter of BON, the Business Object Notation.
+%%%
+%%% This document is written in a literate style using the PVS theorem
+%%% prover. Writing system specifications in this fashion is called
+%%% (formal) domain engineering and system specification. Our formal
+%%% domain model is written in PVS's higher-order logic (HOL). Our
+%%% informal domain model is written in the Free \& Fair System
+%%% Specification Language, or FAFESSL for short. FAFESSL is a
+%%% daughter of Extended BON which was, in turn, a daughter of BON,
+%%% the Business Object Notation.
 %%%
 %%% See our Bibliography project at GitHub,
 %%% \url{https://github.com/FreeAndFair/Bibliography}, for more
 %%% information.
 %%%
+%%% \section{Refinement}
+%%%
 %%% In order to relate PVS to FAFESSL, we must define a refinement
 %%% relationship between their two type systems. Informally, that
 %%% mapping is described in the following paragraph.
 %%%
-%%% We map PVS public theories and their contents to FAFESSL constructs.
-%%% The top-level corla theory maps to the FAFESSL system specification,
-%%% theories map to clusters, (PVS) types map to (FAFESSL) types, and
-%%% functions map to features.  We use special comments to denote
-%%% informal specifications which will be extracted using a shell script
-%%% into a well-typed FAFESSL informal domain model specification.
+%%% We map PVS public theories and their contents to FAFESSL
+%%% constructs.  The top-level corla theory maps to the FAFESSL system
+%%% specification, theories map to clusters, (PVS) types map to
+%%% (FAFESSL) types, and functions map to features.  We use special
+%%% comments to denote informal specifications which will be extracted
+%%% using a shell script into a well-typed FAFESSL informal domain
+%%% model specification.
 %%%
 %%% As mentioned above, we write formal specifications using a
 %%% literate style, a la Knuth. Doing so permits us to document our
 %%% thinking and the system design, from domain modeling and
 %%% engineering all of the way to formal specification and
-%%% verification, in a fashion that produces beautifully printed
-%%% books and interactive hypertext.
+%%% verification, in a fashion that produces beautifully printed books
+%%% and interactive hypertext.
```

Comment:
I'm guessing that the paragraph was automatically re-justified, perhaps along with others nearby, or during proposed edits.
I'd prefer that these sorts of changes be avoided, even if lines are left shorter than possible.